src/pages/editor-page/canvas/canvas.tsx

```diff
@@ -262,8 +264,9 @@ export const Canvas: React.FC<CanvasProps> = () => {
                 </Controls>
                 <MiniMap
                     style={{
-                        width: 100,
-                        height: 100,
+                        width: isMobile ? 60 : 100,
+                        height: isMobile ? 60 : 100,
+                        marginRight: '3.3rem',
```

Comment:
```suggestion
                        width: isMobile ? 60 : 100,
                        height: isMobile ? 60 : 100,
```
I would remove the margin and instead, I would do the controls component position top-center in case of mobile, like this:
`                <Controls
                    position={isMobile ? 'top-center' : 'bottom-center'}
...`